tasks/bundle-api.js

```diff
@@ -21,11 +24,11 @@ async function bundleAPI() {
                 tsconfigOverride: {
                     compilerOptions: {
                         removeComments: true,
-                        target: 'es5',
+                        target: 'ES6',
```

Comment:
This change should not be the part of this PR. We should analyze the real cases, if we should target old browsers and provide polyfills for them, or maybe we should target `esnext`, or maybe there should be multiple targets. Let's leave it for another issue.

Reply:
Okay, open vscode have master branch or something and use the API script :) Poof a error

Reply:
I've fixed this error following the suggestion from the message https://github.com/darkreader/darkreader/pull/2571/files